src/bptl/work_units/kownsl/tasks.py

```diff
@@ -332,3 +297,45 @@ def set_review_request_metadata(task: BaseTask) -> dict:
     )
 
     return {}
+
+
+@register
+def get_approval_toelichtingen(task: BaseTask) -> dict:
+    """
+    Get the "toelichtingen" of all reviewers that responded to the review request.
+
+    **Required process variables**
+
+    * ``kownslReviewRequestId``: the identifier of the Kownsl review request.
+    * ``zaakUrl``: URL reference to the zaak used for the review itself.
+
+    **Sets the process variables**
+
+    * ``toelichtingen``: a string containing the "toelichtingen" of all reviewers.
+    """
+
+    # Get the review request with id as given in variables
+    review_request = get_review_request(task)
+    review_request_id = review_request["id"]
```

Comment:
This isn't even needed, since you can directly get `review_request_id = check_variable(variables, "kownslReviewRequestId")`